frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -299,15 +309,21 @@ class DashboardHeaderContainer extends Component<DashboardHeaderProps> {
       >
         {t`Cancel`}
       </Button>,
-      <ActionButton
+      <Tooltip
```

Comment:
I tried using the Mantine component here but a) it doesn't work out of the box (it's either hidden or positioning if off) and b) we still use `Tooltip`s in the component anyway so it doesn't make a big difference. Will update it later, as planned in the epic.